### DISCUSSION TITLE
2006/07/06/%E7%94%A8%E7%A7%91%E5%AD%A6%E7%9A%84%E6%96%B9%E6%B3%95%E8%BF%BD%E5%A5%B3%E6%9C%8B%E5%8F%8B-%E4%B9%8B-%E5%A5%A2%E4%BE%88%E8%AE%BA

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#90] [Announcements]
# 2006/07/06/%E7%94%A8%E7%A7%91%E5%AD%A6%E7%9A%84%E6%96%B9%E6%B3%95%E8%BF%BD%E5%A5%B3%E6%9C%8B%E5%8F%8B-%E4%B9%8B-%E5%A5%A2%E4%BE%88%E8%AE%BA

有人问我信仰什么，我说我信仰科学。信仰不是随便说说的，要事事以科学为指导才够虔诚，包括追女朋友。前些天我看了一档科技节目，讲的是吸引异性中的一些科学道理。现在听我添油加醋地把其中与奢侈相关的话题总结一番。

https://blog.qizhen.xyz/2006/07/06/%E7%94%A8%E7%A7%91%E5%AD%A6%E7%9A%84%E6%96%B9%E6%B3%95%E8%BF%BD%E5%A5%B3%E6%9C%8B%E5%8F%8B-%E4%B9%8B-%E5%A5%A2%E4%BE%88%E8%AE%BA

**Comment by ruanqizhen** [comment]
之前博客下的留言：

锋
2010年05月1日 11:17 编辑
科学！！

回复
峰.John
2006年10月10日 00:36 编辑
yeah….very gooooood!!
大智慧

回复
奕瑋
2006年07月10日 10:44 编辑
高论已拜读。

回复
Xiaohong
2006年07月10日 09:27 编辑
最后一段写完了连你自己都恶心得要吐了，可见这功夫非同寻常！汗…

回复
Yan
2006年07月9日 20:11 编辑
 
        “当然，如果月薪一万的人能勒紧裤腰带，买辆二手宝马，就算只穿拖鞋，也会有一群小姑娘在后边追着不放的。”
　　
 阮大虾以此为饵，坚持苦蹬拖鞋数载，以明指暗示其不菲的身价和不流于俗的人格~
　　
 

回复
永青
2006年07月7日 10:33 编辑
那些月薪一万，却衣着老土，脚蹬布鞋，捧着本 C++ 的人
这句话有含沙射影之意啊，，阮大侠身边只有两个人平时脚蹬布鞋。。。

回复
Qizhen
2006年07月7日 10:14 编辑
相信可以放言，回我贴的有两类人，一类是读过候捷的书的，另一类没读过。

回复
yi
2006年07月7日 00:32 编辑
引：
“
此文高屋建瓴，一语中的，作者用生动平实的语言将打开mm心扉的钥匙交到读者手中，让人醍醐灌顶，茅塞顿开。相信可以放言，这世界上只有。。。”
 
Speechless…表这么自卑嘛，和楼主有差距不要紧，太自卑往往很难成功的。
总体上，奢侈不奢侈还是体现了物物相易的公平原则。短期看是奢侈；长远看，个人觉得还是等价的。

回复
yoyo
2006年07月6日 20:09 编辑
楼下的嘴不是一般的甜啊，敢问作者，您是如何追到嫂子的呢，快回答问题啊^^

回复
Jing
2006年07月6日 19:56 编辑
此文高屋建瓴，一语中的，作者用生动平实的语言将打开mm心扉的钥匙交到读者手中，让人醍醐灌顶，茅塞顿开。相信可以放言，这世界上只有两种男人，一种是懂得奢侈论的一流男人，比如作者，还有就是不入流的男人，比如 me……
